modules/persistent-storage-byok.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-azure.adoc
+// storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+// storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_content-type: CONCEPT
+[id="byok_{context}"]
+= About "bring your own key" encryption
+
+// TP or GA???
+
+The "Bring your own key" (BYOK) feature encrypts root volumes and the default storage class.
+
+When starting up managed {product-title} clusters, you can pass a custom Key Management Service (KMS) key to the installer, and the installer and `cluster-storage-operator` encrypts the root volumes for the nodes in the cluster, and also applies this key to encrypt the default storage class.
+This allows you to encrypt your owned volumes with your selected key, instead of the platform's default account key.
```

Comment:
```suggestion
This allows you to encrypt storage volumes with your selected key, instead of the platform's default account key.
```
the phrase "your owned volumes" seemed strange to me, but feel free to leave it if you prefer it.